lighthouse-core/audits/byte-efficiency/uses-responsive-images.js

```diff
@@ -112,6 +115,19 @@ class UsesResponsiveImages extends ByteEfficiencyAudit {
     };
   }
 
+
+  /**
+   * @param {LH.Artifacts.ImageElement} image
+   * @return {number};
+   */
+  static computeAllowableWaste(image) {
```

Comment:
nit: `computeAllowableWaste` -> `determineAllowableWaste` (compute implies some non trivial amount of work is being done)